index.html

```diff
@@ -20,38 +20,41 @@
             <li><a href="#">home</a></li>
             <li><a href="#">portfolio</a></li>
             <li><a href="#">blog</a></li>
+            <li>
+              <div class="logo"><img src="src/img/logo.png"  alt=""/></div>
+            </li>
             <li><a href="#">pages</a></li>
             <li><a href="#">about us</a></li>
             <li><a href="#">contact</a></li>
           </ul>
         </header>
-        <div class="header-logo">
-          <img src="src/img/logo.png" />
+
+        <div class="header-headline">
           <p>Expire</p>
         </div>
 
         <div class="header-slider">
           <div class="arrows">
-            <img src="src/img/left.png" class="left-arrow" />
-            <img src="src/img/right.png" class="right-arrow" />
+            <img src="src/img/left.png" class="left-arrow"  alt=""/>
+            <img src="src/img/right.png" class="right-arrow" alt="" />
           </div>
           <p class="slider-headline">
             Expire
           </p>
-          <img src="src/img/line.png" class="line" />
+          <img src="src/img/line.png" class="line"  alt=""/>
           <p class="slider-paragraph">
             Professionaly designed, carefully made for your enjoyement
           </p>
           <div class="slider-buttons">
-            <button class="explore-btn">explore</button>
+            <button class="button">explore</button>
             <button class="learn-btn">
               learn more
             </button>
           </div>
           <div class="slider-dots">
-            <div class="focused"></div>
-            <div class="unfocused"></div>
-            <div class="unfocused"></div>
+            <div class="slider-dots-item-focused"></div>
```

Comment:
I assume that you'r misunderstood my suggestion-_-

There should be something like that:

```
 <div class="slider-dots-item slider-dots-item-focused"></div>
 <div class="slider-dots-item"></div>
 <div class="slider-dots-item"></div>
```

```
.slider-dots-item {
  width: 23px;
  height: 23px;
  border: 1px solid $white;
  ...

  &-focused {
    background: $white;
  }
}
```

In that case you would not repeat pieces of code.

Reply:
Got it, thanks